evm/evmone/Dockerfile

```
@@ -9,7 +9,7 @@ RUN pip3 install durationpy jinja2 pandas
 
 # install evmone
 WORKDIR /root
-RUN git clone --recursive --single-branch --branch bench-evm-codes https://github.com/ewasm-benchmarking/evmone
+RUN git clone --recursive --single-branch --branch v0.4.1 https://github.com/ewasm-benchmarking/evmone
```

If we are using upstream:
```suggestion
RUN git clone --recursive --single-branch --branch v0.4.1 https://github.com/ethereum/evmone
```